internal/telemetry/telemetrystore/store.go

```diff
@@ -0,0 +1,11 @@
+package telemetrystore
+
+import (
+	"github.com/sourcegraph/sourcegraph/internal/database"
+	"github.com/sourcegraph/sourcegraph/internal/telemetry"
+	"github.com/sourcegraph/sourcegraph/internal/telemetry/telemetrystore/teestore"
+)
+
+func NewDefaultStore(exportQueue database.TelemetryEventsExportQueueStore, eventLogs database.EventLogStore) telemetry.EventsStore {
```

Comment:
Aligns with naming of `telemetryrecorder.New`:

```suggestion
func New(exportQueue database.TelemetryEventsExportQueueStore, eventLogs database.EventLogStore) telemetry.EventsStore {
```

Can we also add docstrings here talking about what this is (backing store for `telemetry.Recorder`), and that _most_ callers should _not_ call this? Most callsites should use `telemetryrecorder.New` instead (I think there are other docstrings that look like this you can copy)